packages/core/package.nls.json

```diff
@@ -90,6 +90,7 @@
     "AWS.configuration.description.amazonq": "Amazon Q creates a code reference when you insert a code suggestion from Amazon Q that is similar to training data. When unchecked, Amazon Q will not show code suggestions that have code references. If you authenticate through IAM Identity Center, this setting is controlled by your Amazon Q administrator. [Learn More](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/code-reference.html)",
     "AWS.configuration.description.amazonq.shareContentWithAWS": "When checked, your content processed by Amazon Q may be used for service improvement (except for content processed for users with the Amazon Q Developer Pro Tier). Unchecking this box will cause AWS to delete any of your content used for that purpose. The information used to provide the Amazon Q service to you will not be affected. See the [Service Terms](https://aws.amazon.com/service-terms) for more details.",
     "AWS.configuration.description.amazonq.importRecommendation": "Amazon Q will add import statements with inline code suggestions when necessary.",
+    "AWS.configuration.description.amazonq.workspaceContext": "Allow Amazon Q to create remote workspace for quality improvements.",
```

Comment:
Should this mention how it differs from `workspaceIndex`? Are both needed? Does `workspaceContext` affect `@workspace`, or what does it affect?

Reply:
This will be updated before launch